releases/nightly-build/compose-files/source/docker-compose-nexus-add-security.yml

```diff
@@ -0,0 +1,303 @@
+# /*******************************************************************************
+#  * Copyright 2020 Intel Corporation.
+#  *
+#  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+#  * in compliance with the License. You may obtain a copy of the License at
+#  *
+#  * http://www.apache.org/licenses/LICENSE-2.0
+#  *
+#  * Unless required by applicable law or agreed to in writing, software distributed under the License
+#  * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  * or implied. See the License for the specific language governing permissions and limitations under
+#  * the License.
+#  *
+#  * @author: Leonard Goodell
+#  * EdgeX Foundry, Hanoi, version "master"
+#  * added: May, 2020
+#  *******************************************************************************/
+
+version: '3.7'
+
+volumes:
+  vault-init:
```

Comment:
This file was renamed in my previous PR. Please remove.

Reply:
Done